CHANGELOG.adoc

```diff
@@ -30,6 +31,95 @@ Caused by:
 `quill generate` currently outputs .pem files without an `EC PARAMETERS` section.
 `dfx identity import` will now correctly identify these as EC keys, rather than Ed25519.
 
+=== feat: Remote canister support
+
+It's now possible to specify that a canister in dfx.json references a "remote" canister on a specific network,
+that is, a canister that already exists on that network and is managed by some other project.
+
+Motoko, Rust, and custom canisters may be configured in this way.
+
+This is the general format of the configuration in dfx.json:
+[source, json]
+----
+{
+  "canisters": {
+    "<canister name>": {
+      "remote": {
+        "candid": "<path to candid file to use when building on remote networks>"
+        "id": {
+          "<network name>": "<principal on network>"
+        }
+      }
+    }
+  }
+}
+----
+
+The "id" field, if set for a given network, specifies the canister ID for the canister on that network.
+The canister will not be created or installed on these remote networks.
+For other networks, the canister will be created and installed as usual.
+
+The "candid" field, if set within the remote object, specifies the candid file to build against when
```

Comment:
Can I override a candid setting (e.g., in your ledger example, could I use a different candid file for the local and remote canisters?)

Reply:
Also, could it be useful (not necessarily in this PR) to support arbitrary URLs for the candid file location?